src/frontend/planner_test/tests/testdata/batch_seq_scan.yaml

```diff
@@ -0,0 +1,23 @@
+# This file is automatically generated. See `src/frontend/planner_test/README.md` for more information.
+- id: batch_seq_scan_local
+  name: Test that BatchSeqScan pushed to compute node in local mode
+  sql: |
+    create table t (id int);
+    create materialized view v as select count(*) cnt from t;
+    SET QUERY_MODE TO distributed;
```

Comment:
Currently no `batch_distributed_plan`. May include this feature in some other PR, use this as workaround.